test/test-pages/buzzfeed-1/source.html

```diff
@@ -3550,7 +3550,6 @@ <h1 id="post-title">Student Dies After Diet Pills She Bought Online “Burned He
                                             <div class="sub_buzz_content">
                                             </div>
                                         </div>
-                                        <div class="grid_clear">&nbsp;</div>
```

Comment:
Note regarding this: I had to remove this line from source as it was producing different outputs with jsdom. As we're not maintaining jsdom, I thought it was faire to simply adapt the source so we get consistent results.